model/operation.cc

```diff
@@ -697,6 +697,7 @@ namespace minsky
     description = "\\verb/"+
```

Comment:
Yes - except that description can be altered from the UI. I think we'll need to change description into an Accessor much like we did with rotate, and attach a bb.update() to the setter.